simulator/src/simulator/simulation-node.ts

```diff
@@ -583,6 +584,49 @@ export class SimulationNode {
       })
     })
   }
+
+  /**
```

Comment:
Might be useful to include a short description of how exec is different from spawn in terms of execution runtime.

Reply:
updated the comment to include `exec()` buffering all output and the potential for a buffer overflow